dagshub/data_engine/model/datasource.py

```diff
@@ -61,13 +62,38 @@ class DatapointMetadataUpdateEntry(json.JSONEncoder):
     allowMultiple: bool = False
 
 
+@dataclass
+class Field:
+    column: str
+    as_of_time: Optional[Union[float, datetime.datetime]] = None
+    alias: Optional[str] = None
+
+    @property
+    def as_of_timestamp(self) -> Optional[int]:
+        if self.as_of_time:
```

Comment:
Same suggestion as it was in the `to_dict`:
```suggestion
        if self.as_of_time is not None:
```